packages/react/src/file-upload/docs/overview.mdx

```diff
@@ -43,6 +38,68 @@ File upload is used in forms to enable users to upload files they need. There ar
 - use in modals or page alerts
 - change the default behaviour of the component by adding a standalone upload file button.
 
+## Multiple files
+
+Selecting multiple files is also supported with FileUpload. Simply add `multiple={true}`, and you can select as many files as you want. You can also set `maxFiles` to limit how many files can be selected.
+
+```jsx live
+() => {
+	const [value, setValue] = React.useState();
+	return (
+		<FileUpload
+			label="Upload files"
+			hint="Upload a maximum of three files"
+			value={value}
+			onChange={setValue}
+			multiple={true}
+			maxFiles={3}
```

Comment:
Alphabetise?

```suggestion
			hint="Upload a maximum of three files"
			label="Upload files"
			maxFiles={3}
			multiple={true}
			onChange={setValue}
			value={value}
```

Reply:
So I think all of the docs examples have been "show the new props last so you can see them at a glance"

Reply:
Ahh ok, good to know. I'll try to keep this in mind for when I'm updating docs.